content/docs/dvclive/how-it-works.md

```diff
@@ -109,22 +109,80 @@ with Git, in which case you can use
 
 ## Setup to Run with DVC
 
-You can create or modify the `dvc.yaml` file at the base of your repository (or
-elsewhere) to define a [pipeline](#setup-to-run-with-dvc) to run experiments
-with DVC or
-[customize plots](/doc/user-guide/experiment-management/visualizing-plots#defining-plots).
-A pipeline stage for model training might look like:
+To run experiments with DVC, define a <abbr>DVC pipeline</abbr> using
```

Comment:
It would be great to have an explanation why it is better (to run with DVC)

Reply:
TBH we don't know whether it's better or not :)  It depends on the use case. But it makes sense to give an overview of possible scenarios where DVCLive or DVC is preferred.

Reply:
My rationale here was that it's addressed in many other places (like in `dvclive/index`) and this page is more about the how than the why. Not a blocker for me either way, though, so let me see if I can come up with something to put here.